lib/ht/search_client/property_ids_stay_search.rb

```diff
@@ -72,12 +72,18 @@ def search_options
     end
 
     def allowed_params
-      super + [:property_ids, :from, :to]
```

Comment:
`:from` and `:to` get added to the lot by `StaySearch`, so they are redundant here